assets/sip_samples/3D_3d4bd7ca-38c6-11ed-95f2-7e92631d7d28/data/mets.xml

```diff
@@ -90,23 +90,21 @@
                 ADMID="uuid-e2dcd7c5-5fad-4bcd-a7c7-762b0be75d0f"
                 DMDID="uuid-3936403d-133f-4765-b3b9-0a46df28db17"
             />
-            <div ID="uuid-fa1ecdd5-ecec-488d-baac-01025d996b77" LABEL="Representations">
-                <div ID="uuid-B0D5E486-C582-41BC-BD2D-50543FC897C1" LABEL="representation_1">
-                    <mptr xlink:type="simple"
-                        xlink:href="./representations/representation_1/mets.xml" LOCTYPE="URL" />
-                </div>
-                <div ID="uuid-FDF37384-FEB4-49BF-82EA-513F6185B89F" LABEL="representation_2">
-                    <mptr xlink:type="simple"
-                        xlink:href="./representations/representation_2/mets.xml" LOCTYPE="URL" />
-                </div>
-                <div ID="uuid-0AC25CA1-6802-413C-9E32-AAD6DA2550BF" LABEL="representation_3">
-                    <mptr xlink:type="simple"
-                        xlink:href="./representations/representation_3/mets.xml" LOCTYPE="URL" />
-                </div>
-                <div ID="uuid-6B54B923-2DA0-47DA-82FB-A67206D34BA7" LABEL="representation_4">
-                    <mptr xlink:type="simple"
-                        xlink:href="./representations/representation_4/mets.xml" LOCTYPE="URL" />
-                </div>
+            <div ID="uuid-B0D5E486-C582-41BC-BD2D-50543FC897C1" LABEL="Representations/representation_1">
+                <mptr xlink:type="simple"
+                    xlink:href="./representations/representation_1/mets.xml" LOCTYPE="URL" xlink:title="representation_1" />
+            </div>
+            <div ID="uuid-FDF37384-FEB4-49BF-82EA-513F6185B89F" LABEL="Representations/representation_2">
+                <mptr xlink:type="simple"
+                    xlink:href="./representations/representation_2/mets.xml" LOCTYPE="URL" xlink:title="representation_2" />
+            </div>
+            <div ID="uuid-0AC25CA1-6802-413C-9E32-AAD6DA2550BF" LABEL="Representations/representation_3">
+                <mptr xlink:type="simple"
+                    xlink:href="./representations/representation_3/mets.xml" LOCTYPE="URL" xlink:title="representation_3" />
+            </div>
+            <div ID="uuid-6B54B923-2DA0-47DA-82FB-A67206D34BA7" LABEL="Representations/representation_4">
+                <mptr xlink:type="simple"
+                    xlink:href="./representations/representation_4/mets.xml" LOCTYPE="URL" xlink:title="representation_4" />
```

Comment:
`xlink:title` moet de waarde van de `@ID` van de filegrp van de representatie in de filesec hebben